src/renderers/dom/shared/eventPlugins/__tests__/ChangeEventPlugin-test.js

```diff
@@ -54,7 +54,7 @@ describe('ChangeEventPlugin', () => {
     );
 
     setUntrackedValue(input, true);
-    ReactTestUtils.SimulateNative.click(input);
+    ReactTestUtils.SimulateNative.change(input);
```

Comment:
Radios and Checkboxes were listening for onClick, i removed that in favor of just listening for onChange/onInput like text inputs.

I'm not entirely sure why the original code used click, the comment suggested it was for IE8, it appears to work fine in IE9 though